src/schema/v2/partnerOffer.ts

```diff
@@ -16,6 +18,9 @@ export const PartnerOfferType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       resolve: ({ artwork_id }) => artwork_id,
     },
+    available: {
+      type: GraphQLBoolean,
+    },
```

Comment:
**suggestion (non-blocking):** Minor, but I believe we prefer to rename these upstream booleans to e.g. `isAvailable`

(Examples [here](https://github.com/artsy/metaphysics/blob/836358a9fc9b960ee18ab7ec606ed331bbe488bf/src/schema/v2/me/index.ts#L388-L391) and [here](https://github.com/artsy/metaphysics/blob/836358a9fc9b960ee18ab7ec606ed331bbe488bf/src/schema/v2/gene.ts#L137-L140))

Reply:
thanks, fixed!